### PR TITLE
Loading / asset optimizations

### PR DESCRIPTION
**Pull request by bcjordan**
- In static/craft/Sliced_Parts and static/skins/craft/images:
  `pngquant --skip-if-larger --speed 1 *.png --ext=.opt.png`
  and then manually reviewed and threw out optimized images with perceptible differences
  Changed level 1 image asset loading from ~1.8mb to ~800kb
- Load game music completely after game initialization
  Previously erroneously loaded alongside essential game assets, now properly loads after game asset downloads completed (incl. assets for next level).
- Convert slides .pngs to 460-width .jpgs, update slides.yml to reflect new assets
  `mkdir jpegs; sips -s format jpeg *.* --out jpegnew`
  `sips -Z 460 *.* --out resized`

IE9 re-test in progress

### CODE REVIEW

File: apps/src/craft/craft.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ var characters = {
 var interfaceImages = {
   1: [
     MEDIA_URL + "Sliced_Parts/MC_Loading_Spinner.gif",
-    MEDIA_URL + "Sliced_Parts/Game_Window_BG_Frame.png",
+    MEDIA_URL + "Sliced_Parts/Frame_Large_Plus_Logo.png",
```
Comment (bcjordan) [inline apps/src/craft/craft.js:48]: This asset had changed path, now will pre-load properly

Comment (breville) [inline apps/src/craft/craft.js:48]: deliberate new filename?

**Comment by breville** [comment]
How bad can pngquant artefacts get?  I must admit I never noticed a difference on prior runs.

Love the optimally-packed spritesheets.

lgtm.

**Comment by bcjordan** [comment]
@breville only noticed a few very subtle artifacts on light fade gradients. e.g.:

Un-optimized (33kb):

pop_up_character_steve_neutral

Optimized (a tempting 6kb) (see bottom fade):

`pngquant --skip-if-larger --speed 1 *.png --ext=.opt.png --force`

pop_up_character_steve_neutral opt

**Pull request merged**